lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_git_helper.rb

```diff
@@ -28,23 +28,6 @@ def self.commit_version_bump(include_deliverfile: true, include_metadata: true)
           Fastlane::Helper::GitHelper.commit(message: 'Bump version number', files: files_list, push: true)
         end
 
-        # Calls the `Scripts/localize.py` script in the project root folder and push the `*.strings` files
-        #
-        # That script updates the `.strings` files with translations from GlotPress.
-        #
-        # @env PROJECT_ROOT_FOLDER The path to the git root of the project
-        # @env PROJECT_NAME The name of the directory containing the project code (especially containing the `build.gradle` file)
-        #
-        # @deprecated This method is only used by the `ios_localize_project` action, which is itself deprecated
-        #             in favor of the new `ios_generate_strings_file_from_code` action
-        # @todo [Next Major] Remove this method once we fully remove `ios_localize_project`
```

Comment:
Isn't `ios_localize_project` still used though (not for long, but until we finish our work on the Localization Tooling Modernization and finally migrate all client apps to the new actions)?

Reply:
Ouch! I'm so glad we have code review 😓 

I _did_ run the tests after removing this, but I guess it was a naive of just being happy with that given how old this code is.